app/uk/gov/hmrc/directdebitupdateemailfrontend/config/AppConfig.scala

```diff
@@ -30,8 +27,8 @@ class AppConfig @Inject() (config: Configuration) {
 
   val authTimeoutSeconds: Int = config.get[FiniteDuration]("timeout-dialog.timeout").toSeconds.toInt
   val authTimeoutCountdownSeconds: Int = config.get[FiniteDuration]("timeout-dialog.countdown").toSeconds.toInt
-  def signOutUrl(implicit request: Request[_]): String = BaseUrl.platformHost.getOrElse(config.get[String]("baseUrl.sign-out-local")) +
-    s"/bas-gateway/sign-out-without-state?continue=${routes.SignOutController.signOut.absoluteURL()}"
+  val signOutUrl: String = BaseUrl.platformHost.getOrElse(config.get[String]("baseUrl.sign-out-local")) +
+    s"/bas-gateway/sign-out-without-state?continue=${BaseUrl.ddUpdateEmailFrontend}/direct-debit-verify-email/sign-out"
```

Comment:
what may work is this:
```scala
lazy val signOutUrl: String = BaseUrl.platformHost.getOrElse(config.get[String]("baseUrl.sign-out-local")) +
    s"/bas-gateway/sign-out-without-state?continue=${routes.SignOutController.signOut.url()}"
```
where you use `url` instead of `absoluteUrl` and where it's now a `lazy val`. I think it needs to be `lazy` here cos `url` won't pick up the `/direct-debit-verify-email` part of the url if you don't for some reason

Reply:
whooops I meant
```scala
lazy val signOutUrl: String = BaseUrl.platformHost.getOrElse(config.get[String]("baseUrl.sign-out-local")) +
    s"/bas-gateway/sign-out-without-state?continue=${BaseUrl.ddUpdateEmailFrontend}/${routes.SignOutController.signOut.url()}"
```

Reply:
Thanks Andy, yep I spotted you missed the base url, but the `lazy` fixes the issue I had. Thanks for your help. Will remember that the strange behaviour in testing can be to do with when the value is evaluated.

Reply:
Now done